integrations/nvidia/src/haystack_integrations/components/embedders/nvidia/document_embedder.py

```diff
@@ -31,7 +33,7 @@ class NvidiaDocumentEmbedder:
 
     def __init__(
         self,
-        model: str = "NV-Embed-QA",
+        model: Optional[str] = None,
```

Comment:
We should update the docstring to describe the new behaviour for selecting the default model.